pkg/codegen/golang.go

```diff
@@ -58,61 +53,20 @@ func GenerateGo(pg *PackageGenerator, name string) (buffers map[string]*bytes.Bu
 	}
 	moduleToPackage["meta/v1"] = "meta/v1"
 
-	jsonData, err := ijson.RawMessage(map[string]any{
-		"importBasePath":     "github.com/pulumi/pulumi-kubernetes/sdk/v4/go/kubernetes",
-		"internalModuleName": "utilities",
-		"moduleToPackage":    moduleToPackage,
-		"packageImportAliases": map[string]any{
-			"github.com/pulumi/pulumi-kubernetes/sdk/v4/go/kubernetes/meta/v1": "metav1",
-		},
-	})
-	if err != nil {
-		return nil, fmt.Errorf("failed to marshal language metadata: %w", err)
-	}
-	pkg.Language[langName] = jsonData
-
-	files, err := goGen.GeneratePackage("crd2pulumi", pkg)
+	files, err := goGen.GeneratePackage("crd2pulumi", pkg, nil)
 	if err != nil {
 		return nil, fmt.Errorf("could not generate Go package: %w", err)
 	}
 
-	packageRoot, err := goPackageRoot(pkg.Reference())
-	if err != nil {
-		return nil, fmt.Errorf("could not get package root: %w", err)
-	}
-
 	pkg.Name = oldName
 	delete(pkg.Language, langName)
 
 	buffers = map[string]*bytes.Buffer{}
 	for path, code := range files {
-		newPath, _ := filepath.Rel(name, path)
```

Comment:
Fixing Go code generated in the wrong folder. See #89 
Also needed to ensure tests can find the right folder to compile generated Go code.